apps/agent/app/src/components/useFilteredTransactions.js

```diff
@@ -100,6 +109,7 @@ function useFilteredTransactions({ transactions, tokens }) {
     selectedToken,
     selectedTransactionType,
     symbols,
+    transactionTypes: TRANSACTION_TYPES_LABELS,
```

Comment:
Is this likely to be altered within this hook in the future? It might be simpler to import it directly in `<TransactionFilters/>`

Reply:
Not quite; the reasoning here is to have a single source of truth regarding the filtering and the types. We could import it from `<TransactionFilters/>`, but it'd be weird to have the types coming from one place, and the logic for these from another.